dot/state/grandpa.go

```diff
@@ -568,7 +568,7 @@ func (s *GrandpaState) GetPrevotes(round, setID uint64) ([]types.GrandpaSignedVo
 		return nil, err
 	}
 
-	pvs := []types.GrandpaSignedVote{}
+	var pvs []types.GrandpaSignedVote
```

Comment:
Isn't it a little too much for deepsource to complain about this?

Reply:
Yeah, I don't know what rules we have set up for the project, but this is actually a code smell according to deepsource